sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Specify the release date by replacing the `Unreleased` by the actual date, format is `yyyy-mm-dd`.